cmd/util.go

```diff
@@ -4,13 +4,14 @@ import (
 	"errors"
 	"fmt"
 	"io/ioutil"
-	"os"
 
 	yaml "gopkg.in/yaml.v2"
 
 	"github.com/degica/barcelona-cli/api"
 )
 
+var HeritageConfigFilePath string
+
```

Comment:
I'm guessing this was turned global only so that we can use a different barcelona.yml in the spec, right? Is it instead possible to stub `loadHeritageConfig`? :thinking: maybe it's overkill to include https://github.com/golang/mock just for this PR.

Reply:
Thank you for the reviews 🙏 
You are right. I figured there had to be a better way.
will take a look 👁️

Reply:
@Resonious I looked it and it is a bit of work to do.
if you are ok, I want to go with current implementation 🙏 
I accidentally dismissed your review 😢

Reply:
Hahaha no worries. I'm not too bothered by this.